test/testdata/fixtures/multi-dc-encryption-medusa/k8ssandra.yaml

```diff
@@ -54,7 +54,6 @@ spec:
         thrift_prepared_statements_cache_size_mb: 1
         prepared_statements_cache_size_mb: 1
         slow_query_log_timeout_in_ms: 0
-        block_for_peers_timeout_in_secs: 10000
```

Comment:
Many options were invalid in our e2e tests. They were being filtered out so far, but not anymore.